.github/labels.yml

```diff
@@ -0,0 +1,25 @@
+area:
+  - 'bug'
+  - 'important'
+
+kind:
+  - 'failing-test'
+  - 'cleanup'
+
+priority:
+  - 'low'
+  - 'mid'
+  - 'high'
+
+# File globs for PR labeler
+# refer to github actions/labeler for further documentation
+
+tests:
```

Comment:
I believe we would need to update the glob matches here for use with kyverno and golang. I am also unsure if we want to label the PR with labels like `tests` or `source` based on content paths but I will leave it up to the maintainers to decide.